Makefile

```diff
@@ -82,9 +82,6 @@ race_condition_binary: build
 race_condition_run: race_condition_binary
 	./prometheus.race $(ARGUMENTS)
 
-run: binary
```

Comment:
But `make && ./prometheus` should really be ok if you want a single line,
no? Also it's less surprising to people, as Makefiles are usually not used
to start servers, and people might incorrectly learn that this is the
proper way to run Prometheus.
On May 25, 2015 16:16, "Brian Brazil" <notifications@github.com> wrote:

> In Makefile
> <https://github.com/prometheus/prometheus/pull/728#discussion_r30981874>:
>
> > @@ -82,9 +82,6 @@ race_condition_binary: build
> >  race_condition_run: race_condition_binary
> >  	./prometheus.race $(ARGUMENTS)
> >
> > -run: binary
>
> For the sake of tutorials, I think saving a command is worth it as it
> makes it easier to get things up and running quickly.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/prometheus/prometheus/pull/728/files#r30981874>.
>